### ISSUE TITLE
Cannot Damage entitys

### ISSUE THREAD

**Issue by daditto2012** [Discord-Moder-ator/resentClient#3]
In patch four You cannot damage any entitys (offline HTML) Including player's

**Comment by Elite4r** [comment]
Yeah, that is an issue for singleplayer worlds. If you host a multipalyer server, everything should work as normal.

**Comment by DogeIsKingOfMemes** [comment]
I have this issue too

**Comment by DogeIsKingOfMemes** [comment]
it's very annoying :/

**Comment by Elite4r** [comment]
Resent will most liekly patch this in Resent Client 5.0, this is for the developers to fix.
